demisto_sdk/commands/test_content/test_modeling_rule/init_test_data.py

```diff
@@ -108,8 +107,8 @@ def init_test_data(
                     if not event_log.dataset:
                         event_log.dataset = default_dataset
 
-                    new_mapping = dataset_to_fields_map.get(
-                        event_log.dataset, {}
+                    new_mapping = dict(
+                        sorted(dataset_to_fields_map.get(event_log.dataset, {}).items())
```

Comment:
I think it is better to do the sorting on line 93:

`
                dataset_to_fields_map = {
                    mr.dataset: dict.fromkeys(sorted(mr.fields)) for mr in mr_entity.rules
                }
`